File: B/1398B.py
# solution
def aliceScore() -> int:
    game_str = input()
    count, alice_score = 0, 0
    num_list = []
    for i in range(len(game_str)):
        if (game_str[i] == '1'):
            count+=1
            if(i == len(game_str) -1):
                num_list.append(count)
        else:
            num_list.append(count)
            count = 0
    num_list.sort(reverse=True)
    for i in range (0,len(num_list),2):
        alice_score+=num_list[i]
    
    return alice_score

t = int(input())
for i in range(t):
    print (aliceScore())

File: B/1398B.cpp
#include <bits/stdc++.h>

using namespace std;

//solution function
int aliceMoves() {
    string str; cin >> str;
    vector <int> vec_str;

    int count = 0, alice_score = 0;
    for (int i = 0; i < str.length(); i++) {
        char ch = str[i];
        if (ch == '1') {
            count++;
            if (i == str.length() - 1) vec_str.emplace_back(count);
        } else {
            vec_str.emplace_back(count);
            count = 0;
        }
    }
    sort(vec_str.begin(), vec_str.end(), greater<int>());
    for (int i = 0; i < vec_str.size(); i += 2) {
        alice_score += vec_str[i];
    }
    return alice_score;
}


//main function
int main() {
    int t; cin >> t;
    while (t--) cout << aliceMoves() << "\n";

    return 0;
}